wallets/provider-clover/src/signer.ts

```diff
@@ -3,14 +3,15 @@ import type { SignerFactory } from 'rango-types';
 import { getNetworkInstance, Networks } from '@rango-dev/wallets-shared';
 import { DefaultSignerFactory, TransactionType as TxType } from 'rango-types';
 
+import { CustomSolanaSigner } from './solana-signer.js';
```

Comment:
Wee didn't lazy import `DefaultSolanaSigner` at all. Lazy importing these `CustomSolanaSigner` s will result in small chunks